packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -310,6 +327,73 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     _target.children.add(element);
     element.click();
   }
+
+  Future<String> _getCompressedUri(html.File file, int? imageQuality,
```

Comment:
Extract to a new `ImageResizer` object. Also, try to use: Future<XFile> as the input format for the file, and the output for the resized image.